profiles/api.py

```diff
@@ -0,0 +1,18 @@
+"""
+RESTful viewsets and serializers for user profiles
+"""
+from rest_framework import serializers, viewsets
+from profiles.models import Profile
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    """Serializer for Program objects"""
+    class Meta:  # pylint: disable=missing-docstring
+        model = Profile
+        fields = ('id',)
+
+
+class ProfileViewSet(viewsets.ReadOnlyModelViewSet):
+    """API for the Program collection"""
+    queryset = Profile.objects.all()
+    serializer_class = ProfileSerializer
```

Comment:
is this the REST view? if so it should be in a `views.py` file

I understand what you meant to do, but in the past the `api.py` module contained the api functions used across the app and not the REST APIs, so I would avoid this approach.

Another comment is that we might want to version the rest APIs using directories path like `/api/v0/...`